lib/src/main/kotlin/com/swmansion/starknet/data/types/StarknetString.kt

```diff
@@ -0,0 +1,8 @@
+package com.swmansion.starknet.data.types
+
+import com.swmansion.starknet.data.serializers.StarknetStringSerializer
+import kotlinx.serialization.Serializable
+
+@JvmInline
+@Serializable(with = StarknetStringSerializer::class)
+value class StarknetString(val value: String) : HttpBatchRequestType
```

Comment:
This is fine I suppose, although someone might think `StarknetString` implies something from Cairo i.e. `ByteArray`. 

WDYT about `StringResponse`, `IntResponse`? 

I don't have a strong say on this, just a thought.

Reply:
Together with @THenry14 we had a [discussion](https://github.com/software-mansion/starknet-jvm/pull/461#discussion_r1592379466) about it 😅

Imho, I think that `StarknetString` is a bit far from being confused with `ByteArray`. 

`StringResponse` seems a bit better, but don't you think it might seem to general?